node/service/src/block_sealing.rs

```diff
@@ -251,9 +247,8 @@ pub async fn run_seal_command<B, BI, CB, E, C, TP, SC, CS, CIDP, P>(
 	while let Some(command) = commands_stream.next().await {
 		match command {
 			EngineCommand::SealNewBlock { create_empty, finalize, parent_hash, sender } =>
-				if sealing_mode != SealingMode::Interval ||
-					sealing_create_empty_blocks ||
-					pool.ready().count() > 0
+				// To keep the logs quieter, don't even attempt to create empty blocks unless there are transactions in the pool
+				if create_empty || pool.ready().count() > 0
```

Comment:
This is the important logic change!

Reply:
How is this different from specifying to not create empty blocks?